DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/SendOperation.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Core.Models;
+using System.Threading.Tasks;
+using System.Threading;
+using Speckle.Connectors.Utils.Operations;
+using Speckle.Core.Api;
+using Speckle.Core.Credentials;
+using Speckle.Core.Transports;
+using System.Linq;
+using Autofac;
+
+namespace Speckle.Connectors.Revit.Operations.Send;
+
+public sealed class SendOperation
+{
+  private readonly ILifetimeScope _scope;
```

Comment:
Yeah the intention was that the ISpeckleConverterToSpeckle was the Unit of work and any dependency it required was either Singleton in InstancePerLifetimeScope. It may be that in your case RootObjectBuilder is where the unit of work should live and it would be its dependencies that are new for each instantiation of RootObjectBuilder. I don't know why ISpeckleConverterToSpeckle concrete types would have state, but I can see that legacy dependencies for  RootObjectBuilder might do. We can discuss, but I think I have a note to consider the UoW